sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -187,6 +189,47 @@ class HiveTableScanSuite extends HiveComparisonTest with SQLTestUtils with TestH
     }
   }
 
+  test("SPARK-32867: When explain, HiveTableRelation show limited message") {
+    withSQLConf(HiveUtils.CONVERT_METASTORE_ORC.key -> "false",
+      "hive.exec.dynamic.partition.mode" -> "nonstrict") {
+      withTable("df") {
+        spark.range(1000)
+          .select(col("id"), col("id").as("k"))
+          .write
+          .partitionBy("k")
+          .format("hive")
+          .mode("overwrite")
+          .saveAsTable("df")
+
+        val scan1 = getHiveTableScanExec("SELECT * FROM df WHERE df.k < 3")
+        assert(scan1.simpleString(100).replaceAll("#\\d+L", "") ==
+          "Scan hive default.df [id, k]," +
+            " HiveTableRelation [" +
+            "`default`.`df`," +
+            " org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe," +
+            " Data Cols: [id]," +
+            " Partition Cols: [k]," +
+            " Pruned Partitions: [k=0, k=1, k=2]," +
+            " Statistic: sizeInBytes=8.0 EiB" +
```

Comment:
Printing `8.0 EiB` looks weird and we need `Statistic` here?

Reply:
> Printing `8.0 EiB` looks weird and we need `Statistic` here?

Not so important, IMO, we can remove